lib/database/schemaGenerators.js

```diff
@@ -467,7 +467,7 @@ var AlterTableGenerator = define(null, {
          */
         addForeignKey:function (name, table, opts) {
             opts = opts;
-            if (isArray(name)) {
+            if (Array.isArray(name)) {
```

Comment:
i think there are more of these in this file

Reply:
if not you can remove the isArray var setting at the top of the file

Reply:
I'll go ahead and move them over for consistency. I'm guessing maybe the tests don't hit them or there aren't any issues with them.

Reply:
Fixed in a954338